packages/sdk/src/types/internal/actor.ts

```diff
@@ -33,6 +33,18 @@ export class InternalActor {
         }
     }
 
+    public setAnimationStateEventRaised(animationName: string, state: SetAnimationStateOptions) {
+        if (this.actor) {
+            if (state.enabled !== undefined) {

```

Comment:
Combine conditions

Reply:
Leaving this for now. There may be other sub-conditions coming later.